doc/extending/unittest.txt

```diff
@@ -476,8 +476,10 @@ Here is an example showing how to use ``verify_grad`` on an Op instance:
 .. testcode::
 
     def test_flatten_outdimNone():
-        # Testing gradient w.r.t. all inputs of an op (in this example the op
-        # being used is Flatten(), which takes a single input).
+        """
+        Testing gradient w.r.t. all inputs of an op (in this example the op
+        being used is Flatten(), which takes a single input).
+        """
```

Comment:
This would be better IMO, 
```python
"""
test_flatten_outdimNone

Testing gradient w.r.t. all inputs of an op (in this example the op 
being used is Flatten(), which takes a single input).
"""
```
but I'll stick to the convention that has been used so far.